### PR TITLE
Users/rahul/neustar

### PR DESCRIPTION
**Pull request by rahul0216**
Change(s):
   - Added New Solution Neustar IP GeoPoint, Playbook and Function App based connector. 

   Reason for Change(s):
   -  New Solution

   Version Updated:
   - 1.0.0 - Initial Version

   Testing Completed:
   - Yes
   - End to end deployment testing of Function app and Playbook
   - Functional testing of Playbook

### CODE REVIEW

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+        "postDeployment":  [
+        ],
+        "prerequisitesDeployTemplateFile":  "",
+        "lastUpdateTime":  "2022-08-29T05:32:46Z",
+        "entities":  [
+            "IP"
+        ],
+        "tags":  [
+            "Neustar",
+            "GeoPoint"
```
Comment (anki-narravula) [inline Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json:18]: Please have tag 'Enrichment" also

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Neustar-GetIPGeoInfo",
+        "description":  "When a new sentinel incident is created, this playbook gets triggered and performs the following actions: \n 1. Gets IP Addresses from incident. \n 2. Gets Geographical location information from Neustart IP GeoPoint API. \n 3. Summarize the details and add as a comment to the incident.",
+        "prerequisites":  
+                        "1. Prior to the deployment of this playbook, Neustar IP GeoPoint API Function App Connector needs to be deployed under the same subscription. \n 2. Refer to Neustar IP GeoPoint API Function App Connector documentation to obtain Neustar IP GeoPoint API Key and Shared Secret.",
+        "postDeployment":  [
+        ],
+        "prerequisitesDeployTemplateFile":  "",
```
Comment (anki-narravula) [inline Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json:11]: Add path to function app's azuredeploy.json file

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Once deployment is complete, authorize each connection.
+
+1. Click the Azure Sentinel connection resource
+2. Click edit API connection
+3. Click Authorize
+4. Sign in
+5. Click Save
+6. Repeat steps for other connections
+
```
Comment (anki-narravula) [inline Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/readme.md:39]: Add instructions for how to change the values of URL, Key, Secret etc post deployment (if someone wants to modify)

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json (hunk 1 above)
Comment (rahul0216) [reply Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json]: Added "Enrichment" in tags.

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json (hunk 2 above)
Comment (rahul0216) [reply Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json:11]: Added Function App Name as Parameter.

File: Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/readme.md (hunk 3 above)
Comment (rahul0216) [reply Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/readme.md:39]: Updated readme files with the instructions to change API Url, Key and Secret.

**Review by anki-narravula — APPROVED** [review]

**Pull request merged**